src/inspect_ai/_view/www/src/client/api/types.ts

```diff
@@ -169,6 +170,7 @@ export interface LogViewAPI {
     filename: string,
     filecontents: string | Blob | ArrayBuffer | ArrayBufferView<ArrayBuffer>,
   ) => Promise<void>;
+  download_log: (log_file: string) => Promise<void>;
```

Comment:
This should be `download_log?: (log_file: string) => Promise<void>;` (it shouldn't be required that the clients implement it).